src/components/navigation/navigation.tsx

```diff
@@ -104,99 +91,157 @@ const SocialLink = styled.a`
+ *
+ */
+const SiteNavigation = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  text-align: right;
+  order: 1;
 
   ${up('md')} {
-    display: inline-block;
-    margin-right: 16px;
+    order: 2;
   }
 `;
 
-const LegalLink = styled(Link)`
-  font-weight: 500;
+const SiteNavigationTitle = styled.span`
+  margin-bottom: 16px;
   font-size: 14px;
-  line-height: 22px;
-  color: #202840;
+  font-weight: bold;
+  line-height: 110%;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
+  color: #ffffff;
+`;
+
+const SiteNavigationList = styled.ul`
+  margin: 0;
+`;
+
+const NavigationListItem = styled.li`
+  display: block;
+`;
+
+const SiteNavigationLink = styled(Link)`
+  font-size: 32px;
+  font-weight: bold;
+  line-height: 150%;
+
   text-decoration: none;
+  color: #202840;
+
+  /**
+   * necessary for Safari
+   */
+  &:link {
+    color: #202840;
+  }
+
+  ${up('md')} {
+    font-size: 48px;
+  }
 `;
 
-const Navigation: React.FC = () => (
-  <NavWrapper>
-    <NavBackground />
-    <NavTitle>Nav</NavTitle>
-    <Nav>
-      <NavList>
-        <NavListItem>
-          <NavLink to="/services">Services</NavLink>
-        </NavListItem>
-        <NavListItem>
-          <NavLink to="/clients">Clients</NavLink>
-        </NavListItem>
-        <NavListItem>
-          <NavLink to="/about">About</NavLink>
-        </NavListItem>
-        <NavListItem>
-          <NavLink to="/career">Career</NavLink>
-        </NavListItem>
-        <NavListItem>
-          <NavLink to="/contact">Contact</NavLink>
-        </NavListItem>
-        <NavListItem>
-          <NavLink to="/blog">Blog</NavLink>
-        </NavListItem>
-      </NavList>
-    </Nav>
-    <SocialLinks>
-      <SocialLinkItem>
-        <SocialLink
-          target="_blank"
-          rel="noopener noreferrer"
-          href="https://twitter.com/satellytes_beep"
-        >
-          <IconTwitter />
-        </SocialLink>
-      </SocialLinkItem>
-      <SocialLinkItem>
-        <SocialLink
-          target="_blank"
-          rel="noopener noreferrer"
-          href="https://www.linkedin.com/company/satellytes"
-        >
-          <IconLinkedIn />
-        </SocialLink>
-      </SocialLinkItem>
-      <SocialLinkItem>
-        <SocialLink
-          target="_blank"
-          rel="noopener noreferrer"
-          href="https://www.xing.com/companies/satellytesgmbh"
-        >
-          <IconXing />
-        </SocialLink>
-      </SocialLinkItem>
-    </SocialLinks>
-    <LegalLinks>
-      <LegalLinkItem>
-        <LegalLink to="/imprint">Imprint</LegalLink>
-      </LegalLinkItem>
-      <LegalLinkItem>
-        <LegalLink to="/data-privacy">Data Privacy</LegalLink>
-      </LegalLinkItem>
-    </LegalLinks>
-  </NavWrapper>
+const Navigation: React.FC = (props) => (
+  <NavigationBackground {...props}>
+    <Grid center>
+      <GridItem>
+        <NavigationContent>
+          <MetaContainer>
+            <SocialLinks>
```

Comment:
This could use a separate component or store all that navigation stuff

Reply:
Yep, let's improve it once we are more dynamic with data